Unite.Web/Client/src/domain/donors/components/UploadButton.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <u-upload-dialog
+    v-model="dialogDonors"
+    :subject="'donors'"
+    :templatePathJson="'/templates/donors.json'"
+    :templatePathTsv="'/templates/donors.tsv'"
+    :uploadMethod="uploadDonors"
+  />
+
+  <u-upload-dialog
+    v-model="dialogTreatments"
+    :subject="'treatments'"
+    :templatePathJson="'/templates/treatments.json'"
+    :templatePathTsv="'/templates/treatments.tsv'"
+    :uploadMethod="uploadTreatments"
+  />
+
+  <q-btn
+    label="Upload"
+    title="Upload data"
+    icon="las la-file-upload"
+    color="primary"
+    dense flat no-caps>
+    <q-menu>
+      <q-list>
+        <q-item @click="showUploadDonors" clickable v-close-popup dense>
```

Comment:
show*** is confusing name here. Usually such names like "show", "hide", "display" are associated with display events - whether or not the components should be visible in the screen. ShowUpload* also shows or hides dialog, but I would call it as typical click event with "on" prefix - onUploadDonors, onUploadTreatments. Then inside them we can update display properties for dialogs and actual methods to fetch data from database are uploadDonors, uploadTreatments. So "on*" - event responses on the UI.